minegauler/core/api.py

```diff
@@ -232,8 +232,9 @@ def wrapped(*args, **kwargs):
                 try:
                     getattr(listener, func)(*args, **kwargs)
                 except Exception as e:
+                    write=f"Error occurred calling {func}() on {listener}"
                     self._logger.warning(
-                        f"Error occurred calling {func}() on {listener}"
+                        write
```

Comment:
The proper way to fix this is to do
```python
self._logger.warning(f"Error occurred calling %s() on %s", func, listener)
```
as is already done in `resize_minefield()` for example (your change there should be reverted).

See https://docs.python.org/3/howto/logging.html#logging-variable-data.

Reply:
Okay, i will look into it and resubmit it after correction.

Reply:
I have made the corrections and raised new pr#181